recipes/seadex-genesis/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class SeadexGenesisConan(ConanFile):
+    name = "seadex-genesis"
+    description = "genesis is a generator library developed by Seadex (written in C++11)."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://genesis.seadex.de/"
+    topics = ("generator", "c++")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8.3",
+            "clang": "12",
+            "Visual Studio": "16",
+            "msvc": "192",
+            "apple-clang": "10"
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.options["fmt/*"].header_only = True
+        self.options["spdlog/*"].header_only = True
```

Comment:
Is this really needed? I could build the library removing these, also from seadex-essentials

Reply:
The options "header_only" for the dependencies removed.